src/controllers/trust.individual.beneficial.owner.controller.ts

```diff
@@ -0,0 +1,81 @@
+import { NextFunction, Request, Response } from 'express';
+import * as config from '../config';
+import { logger } from '../utils/logger';
+import { safeRedirect } from '../utils/http.ext';
+import { getApplicationData } from '../utils/application.data';
+import * as CommonTrustDataMapper from '../utils/trust/common.trust.data.mapper';
+import * as PageModel from '../model/trust.page.model';
+
+const INDIVIDUAL_BO_TEXTS = {
+  title: 'Tell us about the individual',
+};
+
+type TrustIndividualBeneificalOwnerPageProperties = {
+  backLinkUrl: string,
+  templateName: string;
+  trustData: {
```

Comment:
better if it will be `pageData` 

also you need pass `trusteeType` because you use it in template

Reply:
Updated